kernel/kernel-util/src/main/java/org/sakaiproject/util/ResourceLoader.java

```diff
@@ -779,6 +774,19 @@ public String toString() {
                        '}';
     }
 
+
+	public static class BundleControl extends ResourceBundle.Control {
+		@Override
+		public List<Locale> getCandidateLocales(String baseName, Locale locale) {
+			List<Locale> candidateLocales = super.getCandidateLocales(baseName, locale);
+
+			if ("".equals(locale.getLanguage())) {
```

Comment:
According to the Javadoc you shouldn't be adding any other `Locale`s when the requested one is `Locale.ROOT`.

Reply:
We use sv_SE as system locale, actual sv_SE_umu to include local swedish translations.
We have en_US and sv_SE_umu as selectable locales in sakai.
If the user selects en_US the locale in sakai is set to "".